pkg/shim/scheduler.go

```diff
@@ -241,16 +242,23 @@ func (ss *KubernetesShim) doScheduling(e *fsm.Event) {
 
 func (ss *KubernetesShim) registerShimLayer() error {
 	configuration := conf.GetSchedulerConf()
+
+	var buildInfoMap map[string]string
+	buildInfoMap["version"] = conf.Version
+	buildInfoMap["date"] = conf.Date
+
 	registerMessage := si.RegisterResourceManagerRequest{
 		RmID:        configuration.ClusterID,
 		Version:     configuration.ClusterVersion,
 		PolicyGroup: configuration.PolicyGroup,
+		BuildInfo:   buildInfoMap,
```

Comment:
This should be fixed as the SI reference has been updated already to include this